.github/workflows/ci-snapshot.yml

```diff
@@ -32,9 +32,10 @@ jobs:
         run:  >
           sudo -E env "PATH=$PATH" bash -c "ulimit -l 65536 && ulimit -a &&
           ./gradlew properties --no-daemon --console=plain -q | grep -E '^version: [\.0-9]+\-SNAPSHOT$' &&
-          ./gradlew --no-daemon --parallel clean check &&
+          (./gradlew --no-daemon --parallel clean check &&
           ./gradlew --no-daemon --parallel sign &&
           ./gradlew --no-daemon --parallel publish"
```

Comment:
Is this trailing `"` still in the right place given the parentheses?

Reply:
Corrected, good catch